packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -20,19 +19,16 @@ import ConditionalWrapper from '../../ConditionalWrapper';
 import { PostTagsPanel } from '../../post/block/PostTagsPanel';
 import { IconSize } from '../../Icon';
 import { LinkWithTooltip } from '../../tooltips/LinkWithTooltip';
-import { useFeature } from '../../GrowthBookProvider';
-import { feature } from '../../../lib/featureManagement';
 
 interface ShareButtonProps {
   onShareClick?: (event: React.MouseEvent, post: Post) => unknown;
   post: Post;
-  copyLinkFeature?: boolean;
 }
 
 function ShareButton(props: ShareButtonProps) {
```

Comment:
Same here, I don't think we should be calling this ShareButton when what it does now is copy. Would confuse devs who has no background to this.